src/store/metadata/stub/content/editions/total-value-of-uk-exports-2022-08.json

```diff
@@ -0,0 +1,130 @@
+      "modified": "2023-08-25T11:09:00",
+      "spatial_resolution": [
+        "N09",
+        "S12",
+        "W06",
+        "E92",
+        "K02"
+      ],
+      "spatial_coverage": "K02000001",
+      "temporal_resolution": [
+        "P1M",
+        "P3M",
+        "P1Y"
+      ],
+      "temporal_coverage": {
+        "start": "2022-09-01T00:00:00Z",
+        "end": "2022-09-30T23:59:00Z"
+      },
+      "versions_url": "https://staging.idpd.uk/datasets/total-value-of-exports/editions/2022-08/versions",
+      "versions": [
+        {
+          "@id": "https://staging.idpd.uk/datasets/total-value-of-exports/editions/2022-08/versions/1",
+          "issued": "2023-05-23T10:14:20"
+        }
+      ],
+      "next_release": "2023-09-24T09:30:00Z",
+      "table_schema": {
+        "columns": [
+          {
+            "name": "period",
+            "datatype": "string",
+            "titles": "Period",
+            "description": "The time period associated with the observation."
+          },
+          {
+            "name": "areacd",
+            "datatype": "string",
+            "titles": "ONS Geography Code",
+            "description": "The geography code associated with the observation."
+          },
+          {
+            "name": "areanm",
+            "datatype": "string",
+            "titles": "Area Name",
+            "description": "The name assiciated with The geography code "
+          },
+          {
+            "name": "geography",
+            "datatype": "string",
+            "titles": "Geography Type",
+            "description": "The categorisation of the geography code"
+          },
+          {
+            "name": "value",
+            "datatype": "decimal",
+            "titles": "Geographic area with Total value of UK exports by at least one provider",
+            "description": "Observational Value"
+          },
+          {
+            "name": "measure",
+            "datatype": "string",
+            "titles": "Measure Type",
+            "description": "Measure Type of the Observational Value"
+          },
+          {
+            "name": "unit",
+            "datatype": "string",
+            "titles": "Unit",
+            "description": "Unit Type of the Observational Value"
+          },
+          {
+            "name": "lower confidence interval (95%)",
+            "datatype": "decimal",
+            "titles": "Lower Confidence Interval",
+            "description": "The interval estimate gives an indication of how much uncertainty there is in our estimate of the true mean, Lower bound"
+          },
+          {
+            "name": "upper confidence interval (95%)",
+            "datatype": "decimal",
+            "titles": "Upper Confidence Interval",
+            "description": "The interval estimate gives an indication of how much uncertainty there is in our estimate of the true mean, Upper bound"
+          },
+          {
+            "name": "observation status",
```

Comment:
obeservation_status, names should be lowercase and spaces replaces with an underscore.